dash/_callback_context.py

```diff
@@ -252,7 +252,11 @@ def timing_information(self):
     def set_props(self, component_id: typing.Union[str, dict], props: dict):
         ctx_value = _get_context_value()
         _id = stringify_id(component_id)
-        ctx_value.updated_props[_id] = props
+        updates = ctx_value.updated_props.get(_id)
+        if updates is not None:
+            ctx_value.updated_props[_id] = {**updates, **props}
```

Comment:
why `{**updates, **props}` rather than `{**props, **updates}`?
```
props = {"left": 1, "right": 2}
updates = {"left": 3}

# updates then props doesn't update
as_written = {**updates, **props}
assert as_written == {"left": 1, "right": 2}

# props then updates does the update
inverting_order = {**props, **updates}
assert inverting_order == {"left": 3, "right": 2}
```

Reply:
I changed the variable `updates` to `existing` for less confusion, we want to merge the latest call onto the existing props.

Reply:
thank you - much appreciated.